spec/lock/lockfile_bundler_1_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-RSpec.describe "the lockfile format", :bundler => "< 2" do
+RSpec.describe "the lockfile format", :bundler => "< 3" do
```

Comment:
This file name no longer makes sense. It should be called `lockfile_bundler_2_spec.rb`